pages/account/statistics.js

```diff
@@ -58,16 +60,34 @@ export async function getServerSideProps(context) {
       }
     );
     data = await res.json();
+
+    let profileSections = [
+      "bio",
+      "links",
+      "milestones",
+      "tags",
+      "socials",
+      "location"
```

Comment:
What about testimonials?

Reply:
Added testimonials. But it could be an optional field. just my opinion.